addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -99,11 +99,7 @@
 
     <!-- Image choosing part of the Media Dialog -->
     <t t-name="wysiwyg.widgets.image">
-        <t t-set="iframeName" t-value="_.uniqueId('fileframe-')"/>
-        <form method="POST"
-            action="/web_editor/attachment/add"
-            enctype="multipart/form-data"
-            t-att-target="iframeName">
+        <form>
```

Comment:
I would have left the action attribute here and read it in JS, but it is the same.